### ISSUE TITLE
model.fit step

### ISSUE THREAD

**Issue by SouthAmericaB** [ZFTurbo/ZF_UNET_224_Pretrained_Model#14]
Hi!
I would like to know how I should perform the model.fit instruction.
model.fit(trainSet, mask_trainSet, batch_size=20, nb_epoch=1, verbose=1,validation_split=0.2, shuffle=True, callbacks=[model_checkpoint])¿? What I write in callback??

And how should I use the weights if I wan't to use pretained weights??

Thank you very much and sorry for the inconvenience!

**Comment by ZFTurbo** [comment]
1) To load weiths use `model.load_weights(your_path)` before fit()
2) You can remove callbacks if you don't use them.

**Comment by SouthAmericaB** [comment]
Thank you very much! 
But I mean that I want to make a first prediction and I haven't got pre trained weights, so what should I do?

Thanks once again!

**Comment by ZFTurbo** [comment]
You can train without pre-trained weights then. It will work without them.

**Comment by SouthAmericaB** [comment]
What I want to do is segment this little region of an medical Image without pre-trained weights.
Data is normalized by dividing (16^-1) because I work with 16 bit images.
My problem is that I get zero segmentation prediction and I can not find the reason.
Could you help me?
Thank you very much!

![image](https://user-images.githubusercontent.com/49905756/59426453-8d9f7200-8dd8-11e9-8134-b34c16dce997.png)
![image](https://user-images.githubusercontent.com/49905756/59426481-a019ab80-8dd8-11e9-8e0f-454bfc95a9be.png)

**Comment by ZFTurbo** [comment]
Did you train your model on some dataset? Model without training won't work on your task.

**Comment by SouthAmericaB** [comment]
Yes! I train my model with "results = model.fit(X_train, Y_train, validation_split=0.2, batch_size=30, epochs=7,  callbacks=[earlystopper, checkpointer, plot_losses])"

**Comment by ZFTurbo** [comment]
There can be many reasons. Check if you multiply your predictions with 255 before draw it.
